docs/admin/metadata/index.rst

```diff
@@ -56,17 +56,26 @@ Hierop zien we:
 * Rechtsboven een knop **Toon logs**. Deze toont de volledige :ref:`audit trail<admin_logging_index>` van de *informatiecategorie*.
 * Rechtsboven een knop **Geschiedenis**. Deze toont de beheer-handelingen die vanuit de beheerinterface zijn uitgevoerd op de *informatiecategorie*.
 * Als de *informatiecategorie* tot de landelijke lijst behoort, linksonder een knop om het scherm te **sluiten** en terug te keren naar de lijst.
-* Als de *informatiecatgeorie* door de organisatie zelf is toegevoegd, linksonder de mogelijkheid om **wijzigingen op te slaan** (indien van toepassing). Er kan voor gekozen worden om na het opslaan direct een nieuwe *informatiecategorie* aan te maken of om direct de huidige *informatiecategorie* nogmaals te wijzigen.
+* Als de *informatiecategorie* door de organisatie zelf is toegevoegd, linksonder de mogelijkheid om **wijzigingen op te slaan** (indien van toepassing). Er kan voor gekozen worden om na het opslaan direct een nieuwe *informatiecategorie* aan te maken of om direct de huidige *informatiecategorie* nogmaals te wijzigen.
 * Rechtsonder de mogelijkheid om de *informatiecategorie* te **verwijderen**.
 
 Op een informatiecategorie zijn de volgende gegevens beschikbaar. Op het scherm worden verplichte velden **dikgedrukt** weergegeven.
 
+**Algemene velden**
+
 * ``Naam``. De naam van de *informatiecategorie*, bijvoorbeeld "convenant".
 * ``Naam meervoud``. De meervoudsvorm van de naam, bijvoorbeeld "convenanten".
 * ``Definitie``. De (mogelijk landelijke) definitie van de *informatiecategorie*.
 * ``UUID``. Een niet-wijzigbaar, automatisch toegekend identificatiekenmerk.
 * ``Oorsprong``. Een niet-wijzigbaar, automatisch toegekende aanduiding van op welke wijze de *informatiecategorie* is toegevoegd; via de landelijke waardelijst of zelf toegevoegd.
 
+**Bewaartermijn**
```

Comment:
```suggestion
**Bewaartermijn**

```
there must be a blank line before lists otherwise they aren't formatted properly